corehq/apps/locations/models.py

```diff
@@ -328,6 +331,21 @@ def get_locations_and_children(self, location_ids):
             include_self=True
         )
 
+    def get_children(self, location_ids):
+        """
+        Takes a set of location ids and returns a django queryset of their children.
+        """
+        return self.get_queryset_descendants(
+            self.filter(location_id__in=location_ids),
+            include_self=False
+        )
+
+    def get_children_ids(self, location_ids):
+        """
+        Takes a set of location ids and returns a django queryset of their children.
+        """
+        return self.get_children(location_ids).location_ids()
+
```

Comment:
These two also feel very specific - I'm wary of making these models too large with things that will only be used in one place.  Also, "children" generally refers to locations whose direct parent is the location in question, whereas these two methods return all descendants.

Reply:
Making that "descendants" sounds right. I was kind of surprised that there was not a method like this already available. Find descendants of a location and not including itself would be a common use case for having such a hierarchical structure.

Can move that if that looks like bloating it, but it is bound to this class so there won't be any better place to put it and would end up just having that function definition everywhere its needed.

Reply:
Tried to trim that down a bit with some refactor.